### PR TITLE
Fix beatmap info wedge not showing up when zero beatmaps are loaded

### LINKED ISSUE #1722: BeatmapInfoWedge not appearing when no BeatmapSets are present
I'm on a fresh install with no BeatmapSets at all and the BeatmapInfoWedge is not showing.

qq

This also happens when you delete the last BeatmapSet (it shows the dummy wedge), leave SongSelect and reenter.

### PR DESCRIPTION
**Pull request by peppy**
Closes #1722.

### CODE REVIEW

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs [hunk 1]
```diff
@@ -66,6 +66,8 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
                 Depth = info?.Depth + 1 ?? 0,
             }, newInfo =>
             {
+                State = beatmap == null ? Visibility.Hidden : Visibility.Visible;
```
Comment (smoogipoo) [inline osu.Game/Screens/Select/BeatmapInfoWedge.cs]: Is this going to conflict with the this.FadeIn() call right below?

Comment (smoogipoo) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs]: Something like _entering_ songselect with no beatmaps?

**Comment by smoogipoo** [comment]
No testcase for this?

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs]: This one is scheduled so takes precedence. Probably don't need the FadeIn below though

**Review by smoogipoo — DISMISSED** [review]

**Comment by smoogipoo** [comment]
Needs conflict resolution.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**